Translations/translation_EN.json

```diff
@@ -1,340 +1,340 @@
+        },
+        "UIMenu": {
+            "text2": [
+                "User",
+                "interface"
+            ],
+            "desc": "User interface settings"
+        },
+        "AdvancedMenu": {
+            "text2": [
+                "Advanced",
+                "settings"
+            ],
+            "desc": "Advanced settings"
+        }
+    },
+    "menuOptions": {
+        "DCInCutoff": {
+            "text2": [
+                "Power",
+                "source"
+            ],
+            "desc": "Set cutoff voltage to prevent battery over-drain. (DC 10V) (S=3.3V per cell, disable PWR limit)"
+        },
+        "SleepTemperature": {
+            "text2": [
+                "Sleep",
+                "temp"
+            ],
+            "desc": "Tip temperature while in \"sleep mode\""
+        },
+        "SleepTimeout": {
+            "text2": [
+                "Sleep",
+                "timeout"
+            ],
+            "desc": "Interval before \"sleep mode\" starts (s=seconds | m=minutes)"
+        },
+        "ShutdownTimeout": {
+            "text2": [
+                "Shutdown",
+                "timeout"
+            ],
+            "desc": "Interval before the iron shuts down (m=minutes)"
+        },
+        "MotionSensitivity": {
+            "text2": [
+                "Motion",
+                "sensitivity"
+            ],
+            "desc": "0=off | 1=least sensitive | ... | 9=most sensitive"
+        },
+        "TemperatureUnit": {
+            "text2": [
+                "Temperature",
+                "unit"
+            ],
+            "desc": "C=Celsius | F=Fahrenheit"
+        },
+        "AdvancedIdle": {
+            "text2": [
+                "Detailed",
+                "idle screen"
+            ],
+            "desc": "Display detailed info in a smaller font on idle screen"
+        },
+        "DisplayRotation": {
+            "text2": [
+                "Display",
+                "orientation"
+            ],
+            "desc": "R=right-handed | L=left-handed | A=automatic"
+        },
+        "BoostTemperature": {
+            "text2": [
+                "Boost",
+                "temp"
+            ],
+            "desc": "Tip temperature used in \"boost mode\""
+        },
+        "AutoStart": {
+            "text2": [
+                "Start-up",
+                "behavior"
+            ],
+            "desc": "O=off | S=heat to soldering temp | Z=standby at sleep temp until moved | R=standby, heat-off until moved"
+        },
+        "CooldownBlink": {
+            "text2": [
+                "Cooldown",
+                "flashing"
+            ],
+            "desc": "Flash temperature reading at idle if tip is hot"
+        },
+        "TemperatureCalibration": {
+            "text2": [
+                "Calibrate",
+                "tip CJC"
+            ],
+            "desc": "Tip-temperature Cold Junction Compensation (not needed if Tip is +/- 5 C of display)"
```

Comment:
°C if the same as before

Reply:
Fixed in [307b6c7](https://github.com/Ralim/IronOS/commit/307b6c76f8d6cb42ad59d5e91729fdce0caee773)